### PR TITLE
Added Delete All Button confirmation

### LINKED ISSUE #470: Delete All Button confirmation
**Describe the solution you'd like**
I clicked by mistake on the button "Delete all" where I wanted to hit the refresh Button. A confirmation or undo feature would be helpful before all messages are deleted.

### PR DESCRIPTION
**Pull request by Tert0**
Closes #470 

Added an Confirmation Dialog to the "delete all" button and fixed tests.

### CODE REVIEW

File: ui/src/tests/message.test.ts [hunk 1]
```diff
@@ -269,6 +274,9 @@ describe('Messages', () => {
     it('deletes all backup messages and navigates to all messages', async () => {
         await navigate('Backup');
         await page.click('#delete-all');
+        await page.waitForSelector(selector.$confirmDialog.selector());
+        await page.click(selector.$confirmDialog.button('.confirm'));
+        await page.waitForSelector('#delete-all:disabled');
```
Comment (jmattheis) [inline ui/src/tests/message.test.ts:279]: This one is duplicated

File: ui/.prettierrc [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "singleQuote": true,
   "trailingComma": "es5",
   "bracketSpacing": false,
-  "jsxBracketSameLine": true,
+  "bracketSameLine": true,
```
Comment (jmattheis) [inline ui/.prettierrc:9]: This also needed fixes in other files.

**Review by jmattheis — APPROVED** [review]
Thanks!

**Comment by Tert0** [comment]
Could you please label this PR as `hacktoberfest-accepted` or label the Repo as `hacktoberfest`.

**Pull request merged**